editorconfig-eclipse-plugin/editorconfig-core-java

```diff
@@ -1 +0,0 @@
-Subproject commit 9ff6e78159540f5c305551237fcf2c7f14701b1d
```

Comment:
@ncjones I'm really sorry I'm not a big expert with git and I don't know how to do that (I'm using EGit and not line command). If it's possible for you to do that, that's cool otherwise I must find time to learn how to do that.